syndeo-template/src/main/java/com/google/cloud/syndeo/transforms/datagenerator/RecordCreator.java

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (C) 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.syndeo.transforms.datagenerator;
+
+import com.google.common.flogger.GoogleLogger;
+import java.util.List;
+import java.util.Random;
+import org.apache.avro.Schema;
+import org.apache.avro.Schema.Field;
+import org.apache.avro.generic.GenericRecord;
+import org.apache.avro.generic.GenericRecordBuilder;
+import org.apache.beam.sdk.schemas.utils.AvroUtils;
+import org.apache.beam.sdk.values.Row;
+import org.apache.beam.sdk.values.Row.FieldValueBuilder;
+
+public final class RecordCreator {
+
+  private static final GoogleLogger logger = GoogleLogger.forEnclosingClass();
+  private static final String ALPHA_NUMBERIC_STRING =
+      "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789abcdefghijklmnopqrstuvxyz";
+
+  public static Random random = new Random();
+
+  public static Row createRowRecord(Schema schema) {
+    org.apache.beam.sdk.schemas.Schema beamSchema = AvroUtils.toBeamSchema(schema);
```

Comment:
let's receive a Beam Schema as input to avoid this conversion

Reply:
The reason to use avro schema is that the field can support custom attribute like size which defines the string field length. Wonder if Beam Schema has the similar support.

Reply:
passed beam schema as parameter